### ISSUE TITLE
Confluence: cql() method is ignoring start parameter for vsn 1.17.2 but not 1.15.7

### ISSUE THREAD

**Issue by bobgott** [atlassian-api/atlassian-python-api#574]
I have run several tests with the Confluence cql method (_version 1.17.1 and 1.17.2_) using a limit of 25 and querying Confluence that has more than 50 pages to return.

```
CQL: space=\'TUPI\' and type=\'page\' and parent=\'644186113\'
start: 0 (second query sets this to 25)
limit: 25
expand: body.storage
```

The first pass through returns the expected 25 pages.  The second pass where the start is set to 25 returns the exact set as the first.  Hence this will go into a loop.

The problem does not appear in version 1.15.7. I don't believe the code changed in confluence.py for those versions hence confused as to why I'm seeing this.

I've spent several rounds testing and I am pretty sure (_or hope_) that the issue is not on my side.

**Comment by gonchik** [comment]
@bobgott Could you share the debug log, please?
You, can enable the logs via next expression
```
import logging
logging.basicConfig(level=logging.DEBUG, format='[%(asctime).19s] [%(levelname)s] %(message)s')
```

**Comment by bobgott** [comment]
Hi @gonchik ... The attached file is logging for the query .. I aborted when it started looping

[logs.txt](https://github.com/atlassian-api/atlassian-python-api/files/5158294/logs.txt)

**Comment by bobgott** [comment]
@gonchik .. I am as perplexed as you.  I was running my tests within PyCharm so I ran them out of PyCharm and the results are the same.  

Attached is the log file with queries from browser (chrome) with a start at 0 and a start at 75. Exact same return.
[logs2.txt](https://github.com/atlassian-api/atlassian-python-api/files/5169915/logs2.txt)

Perhaps its something in my environment (?) .. but its hard to explain why your version 1.15.7 still works.

**Comment by bobgott** [comment]
I did try this app on the newly released 1.17.3 and the problem still persists.  As a side note, I am running python 3.8.5

**Comment by bobgott** [comment]
This turned out to be my error (very sorry).  I created a new environment with ONLY the Atlassian API and the problem did not appear.  One of the libraries that I am using seems to be causing problems with my HTTP queries.  Have not found the culprit yet but the issue is not within the Atlassian api.

**Comment by bobgott** [comment]
closing this issue as it is not related to the api.

**Comment by gonchik** [comment]
@bobgott  I will be happy to know in which situation it works with unexpected behaviour.
Maybe it helps to me for better adjustment

**Comment by bobgott** [comment]
I found the culprit which had to do with my virtual environments.  I'm running [virtualenvwrapper-win](https://github.com/davidmarble/virtualenvwrapper-win/) which sits on top of [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/stable/).  I noticed in the environments I was running .. wrong libraries were being used.  Reinstalled and now I cannot reproduce problem (that's good).

Let's call this issue done with the user tagged as a bozo.
